style/runtimeTemplate.css

```diff
@@ -26,7 +26,54 @@
   margin-top: 10px;
   outline: none;
 }
-
+.create-runtime-radio {
+  display: flex;
+  align-items: center;
+}
+.css-vqmohf-MuiButtonBase-root-MuiRadio-root {
```

Comment:
this isn't going to work at all because the vqmohf id is auto generated, look into using sx for this.

https://mui.com/system/getting-started/the-sx-prop/

Reply:
This is a part of Shared VPC changes which are not in the PR. So removing the unused piece of code